pkg/storage/bucket/s3/config_test.go

```diff
@@ -127,7 +126,7 @@ func TestConfig_Validate(t *testing.T) {
 					SSE:              *sseCfg,
 					SignatureVersion: SignatureVersionV4,
 					StorageClass:     s3_service.StorageClassStandard,
-					STSEndpoint:      mockMinoStsEndpoint(t),
+					STSEndpoint:      "sts.eu-central-1.amazonaws.com",
```

Comment:
that's technically not a valid URL. It should start with a scheme. See [here](https://go.dev/play/p/e29JVtf4AFt?v=) for example

Reply:
true, i was going off our s3 endpoint config testing. I can make this change.

Reply:
fixed.